### DISCUSSION TITLE
Attempting to get openlibm response.

### DISCUSSION THREAD

**Discussion by PowerUser1234** [JuliaLang/julia#47428] [General]
Dear Julia and Team,

I have been trying to get an important, positive response on an issue I have posted at openlibm, a sub library
which is maintained by the Julia team, apparently.  Can they either reply or get on to someone who is more involved
to reply to my message there, kindly, please?

[https://github.com/JuliaMath/openlibm/issues/267](https://github.com/JuliaMath/openlibm/issues/267)

**Comment by PowerUser1234** [comment]
Could someone or those involved with fdlibm please read and consider

[https://github.com/JuliaMath/openlibm/issues/268](https://github.com/JuliaMath/openlibm/issues/268)

kindly, please?

**Comment by DilumAluthge** [comment]
.This repo (the `JuliaLang/julia` repo is for development of the Julia language itself.

If you have bug reports or feature requests for the `JuliaMath/openlibm` repo, those should be posted on that repo, not on the `JuliaLang/julia` repo. However, from glancing at your issues on the `JuliaMath/openlibm`, it seems like they are mostly about Java, not Julia. If you'd like to have a conversation about something not related to Julia, your best bet will be one of the following:
1. The "Offtopic" category of the Julia Discourse: https://discourse.julialang.org/c/offtopic/29
2. The `#random` channel in the Julia Slack: https://julialang.org/slack/
